static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,191 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember} from 'sentry/types';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // NOTE: this is currently used for Github only
+  // TODO(cathy): include snoozing, docs link
+
+  if (!missingMembers?.users) {
+    return null;
```

Comment:
Is there a case where missingMembers is not null and users is? Should that case return an error instead of null?

Reply:
users will at least be an empty array if missingMembers is not null. does this check for that or do i need to explicitly check for the empty array too?

Reply:
oh google says it's not. ok!